src/shipit_uplift/shipit_uplift/__init__.py

```diff
@@ -13,7 +13,7 @@ def create_app(config=None):
     import shipit_uplift.models  # noqa
 
     app = backend_common.create_app(
-        name=shipit_uplift.config.PROJECT_NAME,
```

Comment:
lets change this to

```python
app = backend_common.create_app(
    app_name=shipit_uplift.config.APP_NAME,  # shipit_uplift
    project_name=shipit_uplift.config.PROJECT_NAME,  # shipit-uplift
    ...
)
```

and then ``app.name`` is going to be ``shipit_uplift.config.APP_NAME`` and ``app.project_name`` is going to be ``shipit_uplift.config.PROJECT_NAME``. 

would this be ok with you?

Reply:
:+1: